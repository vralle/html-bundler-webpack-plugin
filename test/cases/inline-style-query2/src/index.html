<!DOCTYPE html>
<html>
  <head>
    <title>Home</title>
    <!-- inline CSS -->
    <link href="./css/index.css?inline" rel="stylesheet" />
    <link href="./css/style.css?inline" rel="stylesheet" />

    <!-- resolve CSS file -->
    <link href="./css/style-file.css" rel="stylesheet" />

    <!-- inline JS -->
    <script src="./js/index.js?inline" defer="defer"></script>
  </head>
  <body>
    <h1>Home</h1>
    <div class="index-css-inline">1</div>
    <div class="style-css-inline">2</div>
    <div class="style-css-file">3</div>
  </body>
</html>