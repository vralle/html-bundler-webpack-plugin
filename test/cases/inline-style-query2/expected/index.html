<!DOCTYPE html>
<html>
  <head>
    <title>Home</title>
    <!-- inline CSS -->
    <style>.index-css-inline {
  color: red;
}
</style>
    <style>.style-css-inline {
  color: green;
}
</style>

    <!-- resolve CSS file -->
    <link href="style-file.e2174e16.css" rel="stylesheet" />

    <!-- inline JS -->
    <script>console.log(">> index.js");</script>
  </head>
  <body>
    <h1>Home</h1>
    <div class="index-css-inline">1</div>
    <div class="style-css-inline">2</div>
    <div class="style-css-file">3</div>
  </body>
</html>